letter_opener.gemspec

```diff
@@ -18,8 +18,12 @@ Gem::Specification.new do |s|
   s.required_rubygems_version = ">= 1.3.4"
   
   if s.respond_to?(:metadata)
-    s.metadata['changelog_uri'] = 'https://github.com/ryanb/letter_opener/blob/master/CHANGELOG.md'
-    s.metadata['source_code_uri'] = 'https://github.com/ryanb/letter_opener/'
-    s.metadata['bug_tracker_uri'] = 'https://github.com/ryanb/letter_opener/issues'
+    s.metadata = {
+      'bug_tracker_uri' = 'https://github.com/ryanb/letter_opener/issues',
```

Comment:
typo :)

Reply:
Thanks so much!

Reply:
Hope  you can squash this, if not, tell me, and I'll squash locally.

Reply:
Please do :)

Reply:
OK, there you are.